compiler-plugin/src/main/java/io/ballerina/stdlib/websocket/plugin/WebSocketServiceValidator.java

```diff
@@ -39,7 +41,11 @@ public void validate() {
         classDefNode.members().stream().filter(child -> child.kind() == SyntaxKind.OBJECT_METHOD_DEFINITION
                 || child.kind() == SyntaxKind.RESOURCE_ACCESSOR_DEFINITION).forEach(methodNode -> {
             FunctionDefinitionNode functionDefinitionNode = (FunctionDefinitionNode) methodNode;
-            if (functionDefinitionNode.qualifierList().get(0).text().equals(PluginConstants.REMOTE_KEY_WORD)) {
```

Comment:
Shall we extract this logic into a separate function so that it is more readable ?

Reply:
This line is removed. And the new one is updated with Thisaru's suggestion. So I think there is no need of moving that to a separate function as it is more readable now.

Reply:
Since this is a lambda-expression I do not think this is the best way to do it. Usually what we expect from a lambda-expression is to execute a relatively small logic, if the logic is complex we could use a separate function.